src/database/mongo/schemas/UserSchema.js

```diff
@@ -6,6 +6,12 @@ const BlacklistedSchema = new Schema({
   blacklister: { type: String, required: true }
 })
 
+const UserConnection = new Schema({
+  name: String,
+  tokens: { type: Object, default: {} },
+  config: { type: Object, default: {} }
```

Comment:
NO DEFAULTS! THANK YOU!

Reply:
IT WASNT WORKING SO I DID THAT BUT NOW I FIGURED OUT THAT IT WASNT IT TAOK